packages/flutter/lib/src/gestures/events.dart

```diff
@@ -232,6 +233,8 @@ abstract class PointerEvent with Diagnosticable {
   }) : localPosition = localPosition ?? position,
        localDelta = localDelta ?? delta;
 
+  /// Corresponds to the motion event that resulted in this pointer event. (only android)
```

Comment:
Can we document this in a more platform-independent way so we can re-use this for the other platforms as well?

Exact wording would depend on the answer to the question below. Does this id uniquely identify the event across framework and engine? Or (if there are multiple PointerEvent with this id) does it identify the native event that generated this framework-side PointerEvent uniquely on the engine side? Something like this should be included in the doc comment.

After that, we can also include a platform-specific paragraph that on android this maps to the ID of the underlying MotionEvent. Or something like that.

Reply:
I will try to address this and the comment below here: 

What do you think of `embedderPointerEventIdentifier` as a possible name for this? Given that this is different from `pointerIdentifier` -- which is used for hit testing, whereas `embedderPointerEventIdentifier` is used to tie back the current pointer event to the platform equivalent which resulted in this event.

If you agree with that, I can elaborate the dart-doc to explain the intent of this field along with the differences between this and `pointerIdentifier`.

Reply:
Can we just shorten this to `embedderId`?

Reply:
The doc should still include the explanation how it is different from the other IDs in this class.

Reply:
SG